framework/src/problems/FEProblemBase.C

```diff
@@ -3506,6 +3506,11 @@ FEProblemBase::prepareMaterials(SubdomainID blk_id, const THREAD_ID tid)
 
   setActiveElementalMooseVariables(needed_moose_vars, tid);
   setActiveMaterialProperties(needed_mat_props, tid);
+
+  // update material data for tracking active material properties
```

Comment:
shouldnt these 3 be nested under the call above?

Reply:
Yes.